packages/sanity/src/core/studio/components/navbar/search/components/filters/filter/inputs/asset/Asset.tsx

```diff
@@ -102,13 +103,15 @@ export function SearchFilterAssetInput(type?: AssetType) {
 
     const AssetSourceComponent = selectedAssetSource?.component
 
-    const fontSize = fullscreen ? 2 : 1
-
     const buttonText = t(value ? 'search.filter-asset-change' : 'search.filter-asset-select', {
       context: type,
     })
 
-    const accept = get(type, 'options.accept', type === 'image' ? 'image/*' : '')
+    const accept = get(
```

Comment:
oh wow, TIL you can search by uploading an image

Reply:
Makes two of us!